binder/postBuild

```diff
@@ -1,6 +1,6 @@
 #Download Apache Spark
-wget https://archive.apache.org/dist/spark/spark-3.3.2/spark-3.3.2-bin-hadoop3.tgz
-tar -xzf spark-3.3.2-bin-hadoop3.tgz
+wget https://archive.apache.org/dist/spark/spark-3.4.0/spark-3.4.0-bin-hadoop3.tgz
+tar -xzf spark-3.4.0-bin-hadoop3.tgz
 
```

Comment:
Please add the following code

```
hadoop_s3_version=3.3.4
aws_sdk_version=1.12.402

curl https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/${hadoop_s3_version}/hadoop-aws-${hadoop_s3_version}.jar -o $HOME/spark-3.4.0-bin-hadoop3/jars/hadoop-aws-${hadoop_s3_version}.jar

curl https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/${aws_sdk_version}/aws-java-sdk-bundle-${aws_sdk_version}.jar -o $HOME/spark-3.4.0-bin-hadoop3/jars/aws-java-sdk-bundle-${aws_sdk_version}.jar
```

Reply:
yes I removed this to test if this was causing any issues with the binder. Added them back